src/chains/filecoin/options/tsconfig.json

```diff
@@ -15,6 +15,10 @@
     {
       "name": "@ganache/options",
       "path": "../../../packages/options"
+    },
+    {
+      "name": "@ganache/utils",
+      "path": "../../../packages/utils"
     }
   ]
 }
```

Comment:
Does Ganache care about eof newlines?

Reply:
Generally, yes :)

Reply:
Done - it only writes changes if they are materially different, so they will get newlines as, and when we change their contents.